pkg/controller/operators/catalog/operator.go

```diff
@@ -56,16 +56,17 @@ var timeNow = func() metav1.Time { return metav1.NewTime(time.Now().UTC()) }
 // resolving dependencies in a catalog.
 type Operator struct {
 	*queueinformer.Operator
-	client            versioned.Interface
-	lister            operatorlister.OperatorLister
-	namespace         string
-	sources           map[resolver.CatalogKey]resolver.SourceRef
-	sourcesLock       sync.RWMutex
-	sourcesLastUpdate metav1.Time
-	resolver          resolver.Resolver
-	subQueue          workqueue.RateLimitingInterface
-	catSrcQueueSet    queueinformer.ResourceQueueSet
-	reconciler        reconciler.ReconcilerFactory
+	client                versioned.Interface
+	lister                operatorlister.OperatorLister
+	namespace             string
+	sources               map[resolver.CatalogKey]resolver.SourceRef
+	sourcesLock           sync.RWMutex
+	sourcesLastUpdate     metav1.Time
+	resolver              resolver.Resolver
+	subQueue              workqueue.RateLimitingInterface
+	catSrcQueueSet        queueinformer.ResourceQueueSet
+	namespaceResolveQueue workqueue.RateLimitingInterface
```

Comment:
Fix the unit test panic by adding this to the test setup in `subscriptions_test.go`:
```go
o.namespaceResolveQueue = workqueue.NewRateLimitingQueue(workqueue.DefaultControllerRateLimiter())
```